pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -76,12 +103,6 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		wc.Resources.CpuMaximum = cpuMaximum
 	}
 
-	cpuShares := milliCPUToShares(cpuLimit.MilliValue(), isolatedByHyperv)
```

Comment:
this means when only cpuRequest is set, it won't take any effect anymore. Should we keep it as is?

Reply:
Yes, that's intended. Reserve still works for scheduling but has no impact on runtime. If we set shares, then the limit is ignored by Docker today